ssv/value_check.go

```diff
@@ -96,11 +98,19 @@ func ProposerValueCheckF(
 			return errors.Wrap(err, "duty invalid")
 		}
 
-		if blockData, err := cd.GetBellatrixBlindedBlockData(); err == nil {
-			return signer.IsBeaconBlockSlashable(sharePublicKey, blockData.Slot)
+		if blockData, _, err := cd.GetBlindedBlockData(); err == nil {
```

Comment:
I would do something like this 

```go
type blockSubset interface { // if some other interface exist this isn't needed
   Slot() (slot, error)
   IsBeaconBlockSlashable() error
}
var blockData blockSubset
var err error
blockData, _, err = cd.GetBlindedBlockData()
if err != nil {
   blockData, err = cd.GetBlockData()
   if err != nil {
       return err // neither type of block works?
   }
}
slot, serr := blockData.Slot()
if serr != nil {
  return errors.Wrap(serr, "failed to get slot from block data")
}
return signer.IsBeaconBlockSlashable(sharePublicKey, slot)
}
```

this is just code-style and trying to make code more generic and less repeating yourself. its up to you if you prefer existing version just keep it.

Reply:
noted